packages/E2ETest/wdio/test/VisitAllPages.spec.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+import { BasePage, By } from '../pages/BasePage';
+//import HomePage from '../pages/HomePage';
+
+//import assert from 'assert';
+
+let pages = [
+  '<ActivityIndicator>',
+  '<Button>',
+  '<CheckBox>',
+  //  'Custom Views',
+  '<DatePicker>',
+  'Fast Path Texts',
+  '<FlatList>',
+  '<Flyout>',
```

Comment:
New test pages get added/deleted somewhat often. If we keep a whitelist of pages to visit, the list could become out of sync with actual pages.

How much harder would it be to implement this as a blacklist instead of a whitelist?

Reply:
I agree and have been looking for a more data-driven way to select the pages to load, but haven't found one yet.  Surprisingly, there isn't a simple Children API available on the WebDriver Element interface.  There appears to be a way to provide a custom selector, which I think I may end up using for the next phase of tests (doing tree dumps).  I'll research this some more and likely address this in a follow-up change.